src/OpenSSL-standalone.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="OpenSSL-standalone" name="OpenSSL License - standalone">
+      <crossRefs>
+         <crossRef>http://www.openssl.org/source/license.html</crossRef>
```

Comment:
Found a couple of cross-references to this one as a standalone:

https://library.netapp.com/ecm/ecm_download_file/ECMP1196395

https://hstechdocs.helpsystems.com/manuals/globalscape/archive/cuteftp6/open_ssl_license_agreement.htm